### PR TITLE
[4] s/Toolbar/array

### PR DESCRIPTION
**Pull request by PhilETaylor**
s/Toolbar/array

Clearly not a Toolbar, but is clearly an array

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on f0af243782884e21a4322084a9c0dfa7f9aebe44<br /><br />code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33012">issues.joomla.org/tracker/joomla-cms/33012</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on f0af243782884e21a4322084a9c0dfa7f9aebe44<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33012">issues.joomla.org/tracker/joomla-cms/33012</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33012">issues.joomla.org/tracker/joomla-cms/33012</a>.</sub>

File: libraries/src/Toolbar/Toolbar.php [hunk 1]
```diff
@@ -70,7 +70,7 @@ class Toolbar
 	/**
 	 * Stores the singleton instances of various toolbar.
 	 *
-	 * @var    Toolbar
+	 * @var    array
```
Comment (HLeithner) [inline libraries/src/Toolbar/Toolbar.php]:
```suggestion
	 * @var    Toolbar[]
```

Not sure if we use this syntax

Comment (wilsonge) [reply libraries/src/Toolbar/Toolbar.php]: We do

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**